MIGRATION.md

```diff
@@ -155,6 +153,8 @@ If `oTypographyItalic` has been used without one of the three type mixins `oTypo
 + @include oTypographyDisplay($style: 'italic', $include-font-family: false);
 ```
 
+Note: If your project uses `oTypographyItalic` once to style multiple elements with different font families, you may use the css property `font-style: italic` directly instead. We don't recommend this approach because you will not be warned if the italic font required does not exist.
```

Comment:
This note confused me, how come we have a note to mention an alternative solution that we do not recommend?

Reply:
Maybe I should remove it and help should the question come up. Users may have done something like this, which could be applied to any element of any font-family:

```scss
.italic {
    @include oTypographyItalic(); // `font-style: italic`
}
```

It would be totally acceptable to replace `@include oTypographyItalic();` with just `font-style: italic` to help along a complex migration (nothing new would break). But as we don't provide an italic font-face for each font and weight it's not ideal -- the `italic` class could be used with other classes in any combination and mistakenly produce a browser rendered faux italic.

Reply:
Okay, that explanation and example  made more sense than the note did on its own.

Maybe add some of that explanation/example to the note or as you said remove the note and help someone if this does come up.

Reply:
Maybe:

>Note: If it aids your projects migration you may swap `oTypographyItalic` for the css property `font-style: italic` directly instead. We don't recommend this approach because you will not be warned if the italic font required does not exist.

Reply:
I like that!